dsa-solutions/lc-solutions/0500-0599/0540-single-element-in-sorted-array.md

```diff
@@ -0,0 +1,184 @@
+---
+id: single-element-in-sorted-array
+title: Single Element in a Sorted Array
+sidebar_label: Single Element in Sorted Array
+tags: 
+    - Binary Search
+    - Algorithm
+    - O(log n)
+---
+
+## Problem Description
+
+| Problem Statement                                       | Solution Link                                                              | LeetCode Profile                                        |
+| :------------------------------------------------------ | :------------------------------------------------------------------------- | :------------------------------------------------------ |
+| [Single Element in a Sorted Array](https://leetcode.com/problems/Single Element in a Sorted Array/description/) | [Single Element in a Sorted Array Solution on LeetCode](https://leetcode.com/problems/Single Element in a Sorted Array/solutions/) | [Nikita Saini](https://leetcode.com/u/Saini_Nikita/) |
```

Comment:
update or put correct link because your link `https://leetcode.com/problems/Single Element in a Sorted Array/solutions/` is not correct.